lib/services/workflow-api-service.js

```diff
@@ -155,7 +236,22 @@ function workflowApiServiceFactory(
     };
 
     WorkflowApiService.prototype.findActiveGraphForTarget = function(target) {
-        return taskGraphStore.findActiveGraphForTarget(target);
+        return waterline.graphobjects.findOne({
+            node: target,
+            _status: Constants.Task.ActiveStates
+        });
+    };
+
+    WorkflowApiService.prototype.getActiveWorkflows = function(query) {
```

Comment:
I see what you mean now ().  The problem is that this function returns all workflows that match the provided query, not just active workflows.  Again, I think most of these functions need to be renamed to follow a consistent convention.